src/pages/VeSDL/index.tsx

```diff
@@ -347,6 +356,9 @@ export default function VeSDL(): JSX.Element {
                 </Typography>
               </div>
               <DatePicker
+                disabled={
+                  !veSdlTokenBalance.isZero() && sdlTokenInputValueBn.gt(Zero)
```

Comment:
@hammeiam @nkrulikovsky can you dbl check this logic?

Reply:
I think date picker can be disable when both `veSdlTokenBalance `and `sdlTokenInputValueBn `are `zero`.

Reply:
Logic is correct as David suggested.
It prevents user to input SDL amount and Date input simultaneously as @hammeiam suggested.